SalesforceCommerceCloud/pwa-kit
# [App Extensibility ⚙️] Remove `extends` key from pwa-kit-create-app (@W-17373279@)

# Description



This PR removes the obsolete `extend` key from the App Generator and the rest of the files in the monorepo. 


# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Removed: `EXTENSIBILITY_QUESTIONS`, `ccExtensibility` object usages, `retail-app-ext` and `retail-app-no-ext` presets.

# How to Test-Drive This PR

- Generate a new project and verify the package.json file does not include the "extend" key.
- Verify the "extend" key has been removed from any packages in the monorepo.

# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
